eng/versioning/version_client.txt

```diff
@@ -434,8 +434,8 @@ com.azure.tools:azure-sdk-build-tool;1.0.0;1.1.0-beta.1
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
 # In the pom, the version update tag after the version should name the unreleased package and the dependency version:
 # <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
-unreleased_com.azure:azure-core;1.42.0
-unreleased_com.azure:azure-core-http-netty;1.13.6
+unreleased_com.azure:azure-core;1.43.0-beta.1
+unreleased_com.azure:azure-core-amqp;2.9.0-beta.5
```

Comment:
Do we need these two lines? All changes that were in core and amqp were released last week.

Reply:
Good catch, we can remove this, updating the pr.